pom.xml

```diff
@@ -58,8 +60,172 @@
+                    <dockerFile>
+                      ${project.build.directory}/${project.artifactId}-${project.version}/docker-embedded/docker/eip-erpnext-openmrs/Dockerfile</dockerFile>
+                    <contextDir>
+                      ${project.build.directory}/${project.artifactId}-${project.version}/distro</contextDir>
+                  </build>
+                </image>
+                <image>
+                  <name>mekomsolutions/ozone-embedded-eip-odoo-openmrs:${project.version}</name>
+                  <build>
+                    <dockerFile>
+                      ${project.build.directory}/${project.artifactId}-${project.version}/docker-embedded/docker/eip-odoo-openmrs/Dockerfile</dockerFile>
+                    <contextDir>
+                      ${project.build.directory}/${project.artifactId}-${project.version}/distro</contextDir>
+                  </build>
+                </image>
+                <image>
+                  <name>mekomsolutions/ozone-embedded-eip-openmrs-senaite:${project.version}</name>
+                  <build>
+                    <dockerFile>
+                      ${project.build.directory}/${project.artifactId}-${project.version}/docker-embedded/docker/eip-openmrs-senaite/Dockerfile</dockerFile>
+                    <contextDir>
+                      ${project.build.directory}/${project.artifactId}-${project.version}/distro</contextDir>
+                  </build>
+                </image>
+              </images>
+              <buildx>
+                <platforms>
+                  <platform>linux/amd64</platform>
+                  <platform>linux/arm64</platform>
+                </platforms>
+              </buildx>
+            </configuration>
+            <executions>
+              <execution>
+                <id>Build embedded docker images</id>
+                <goals>
+                  <goal>build</goal>
+                </goals>
+                <phase>package</phase>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-resources-plugin</artifactId>
+            <executions>
+              <execution>
+                <!-- Copy Docker Embedded compose file to run directory -->
+                <id>Copy Docker Embedded compose file</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>
+                    ${project.build.directory}/${project.artifactId}-${project.version}/run/docker</outputDirectory>
+                  <overwrite>true</overwrite>
+                  <resources>
+                    <resource>
+                      <directory>
+                        ${project.build.directory}/${project.artifactId}-${project.version}/docker-embedded/docker</directory>
```

Comment:
Isn't a bit weird to have `docker-embedded/**docker**`? Maybe better to have the resources directly in the docker-embedded/ folder?